animal-options.lua

```diff
@@ -0,0 +1,175 @@
+--@ module = true
+
+local overlay = require('plugins.overlay')
+local widgets = require('gui.widgets')
+local gui = require('gui')
+
+local YES = 'Yes'
+local NO = 'No'
+
+-- Make sure an animal unit of your civ is selected as well as not a pet
```

Comment:
why not a pet? pets are still geldable

Reply:
I started with just butchering and forgot to account for gelding. Added `enabled` fields to butcher and adopt to make sure not a pet.